presto-hive/src/main/java/io/prestosql/plugin/hive/HiveModule.java

```diff
@@ -116,7 +116,6 @@ public void configure(Binder binder)
         pageSourceFactoryBinder.addBinding().to(RcFilePageSourceFactory.class).in(Scopes.SINGLETON);
 
         Multibinder<HiveFileWriterFactory> fileWriterFactoryBinder = newSetBinder(binder, HiveFileWriterFactory.class);
-        binder.bind(OrcFileWriterFactory.class).in(Scopes.SINGLETON);
         newExporter(binder).export(OrcFileWriterFactory.class).withGeneratedName();
```

Comment:
i guess you want to remove this line too

Reply:
... or maybe not. Please verify JMX stats are still exported.

Reply:
Yes, it needs to be removed or you get:

```
presto-worker_1_5b6ddd686299 | 1) Explicit bindings are required and io.prestosql.plugin.hive.orc.OrcFileWriterFactory is not explicitly bound.
presto-worker_1_5b6ddd686299 |   while locating io.prestosql.plugin.hive.orc.OrcFileWriterFactory
presto-worker_1_5b6ddd686299 |   at org.weakref.jmx.guice.GuiceMBeanExporter.<init>(GuiceMBeanExporter.java:40)
presto-worker_1_5b6ddd686299 |   at org.weakref.jmx.guice.MBeanModule.configure(MBeanModule.java:39)
presto-worker_1_5b6ddd686299 |   while locating org.weakref.jmx.guice.GuiceMBeanExporter
presto-worker_1_5b6ddd686299 | 
presto-worker_1_5b6ddd686299 | 1 error
presto-worker_1_5b6ddd686299 | com.google.inject.CreationException: Unable to create injector, see the following errors:
presto-worker_1_5b6ddd686299 | 
presto-worker_1_5b6ddd686299 | 1) Explicit bindings are required and io.prestosql.plugin.hive.orc.OrcFileWriterFactory is not explicitly bound.
presto-worker_1_5b6ddd686299 |   while locating io.prestosql.plugin.hive.orc.OrcFileWriterFactory
presto-worker_1_5b6ddd686299 |   at org.weakref.jmx.guice.GuiceMBeanExporter.<init>(GuiceMBeanExporter.java:40)
presto-worker_1_5b6ddd686299 |   at org.weakref.jmx.guice.MBeanModule.configure(MBeanModule.java:39)
presto-worker_1_5b6ddd686299 |   while locating org.weakref.jmx.guice.GuiceMBeanExporter
presto-worker_1_5b6ddd686299 | 
presto-worker_1_5b6ddd686299 | 1 error
presto-worker_1_5b6ddd686299 | 	at com.google.inject.internal.Errors.throwCreationExceptionIfErrorsExist(Errors.java:543)
presto-worker_1_5b6ddd686299 | 	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:186)
presto-worker_1_5b6ddd686299 | 	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:109)
presto-worker_1_5b6ddd686299 | 	at com.google.inject.Guice.createInjector(Guice.java:87)
presto-worker_1_5b6ddd686299 | 	at io.airlift.bootstrap.Bootstrap.initialize(Bootstrap.java:245)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.plugin.hive.HiveConnectorFactory.create(HiveConnectorFactory.java:122)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.connector.ConnectorManager.createConnector(ConnectorManager.java:328)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.connector.ConnectorManager.createCatalog(ConnectorManager.java:198)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.connector.ConnectorManager.createCatalog(ConnectorManager.java:190)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.connector.ConnectorManager.createCatalog(ConnectorManager.java:176)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.metadata.StaticCatalogStore.loadCatalog(StaticCatalogStore.java:88)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.metadata.StaticCatalogStore.loadCatalogs(StaticCatalogStore.java:68)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.server.PrestoServer.run(PrestoServer.java:127)
presto-worker_1_5b6ddd686299 | 	at io.prestosql.server.PrestoServer.main(PrestoServer.java:70)
```